kia-cli/src/main/kotlin/com/faforever/ice/KiaApplication.kt

```diff
@@ -53,8 +53,17 @@ class KiaApplication : Callable<Int> {
     )
     private var telemetryServer: String = ""
 
+    @Option(names = ["--allow-host"], description = ["???"])
+    private var allowHost: Boolean = true
+
+    @Option(names = ["--allow-reflexive"], description = ["???"])
+    private var allowReflexive: Boolean = true
+
+    @Option(names = ["--allow-relay"], description = ["???"])
+    private var allowRelay: Boolean = true
+
     override fun call(): Int {
-        val iceOptions = IceOptions(userId, userName, gameId, forceRelay, lobbyPort, gpgnetPort, telemetryServer)
+        val iceOptions = IceOptions(userId, userName, gameId, forceRelay, lobbyPort, gpgnetPort, telemetryServer, allowHost, allowReflexive, allowRelay)
```

Comment:
The cli args should stay compatible with the current ICE adapter for now. I think there is a force relay option, which should these 3 options accordingly. (You'll need to lookup the old code)

Reply:
uhh, I think you are missing an improvent verb here

Reply:
Oh, I see the java adapter just sets these three depending on the force relay option